docs/web3-eth.rst

```diff
@@ -1096,8 +1096,8 @@ Example
 
 .. code-block:: javascript
 
-    var Tx = require('ethereumjs-tx');
-    var privateKey = new Buffer('e331b6d69882b4cb4ea581d88e0b604039a3de5967688d3dcffdd2270c0fd109', 'hex')
+    var Tx = require('ethereumjs-tx').Transaction;
+    var privateKey = new Buffer.from('e331b6d69882b4cb4ea581d88e0b604039a3de5967688d3dcffdd2270c0fd109', 'hex')
```

Comment:
You don't need `new` here. `Buffer.from(..)` should work.

Reply:
i am still getting the error. 
error: {"code":-32000,"message":"invalid sender"}

Reply:
issue fix. I just specified 'chain'  parameter while creating transaction and it works.
const tx = new Tx(txObject, {'chain':'ropsten'})